viewing_party/party.py

```diff
@@ -1,23 +1,226 @@
 # ------------- WAVE 1 --------------------
+import json
+from unittest import result
 
 def create_movie(title, genre, rating):
-    pass
+    '''
+    Input: movie title, genre, and rating
+    Ouput: If values truthy, produces a dictionary of movies with title, genre, and rating key-value pairs.
+    If inputs are falsy, function returns None.
+    '''
+    movie_dict = {}
+    # if title == True and genre == True and rating == True:
+    if True: 
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+    if title == None or genre == None or rating == None:
+        return None
+    return movie_dict
+
+def add_to_watched(user_data, movie):
+    '''
+    Input: user_data is a dictionary with a key "watched" and value of list of movie dictionaries the user has watched
+    Output: Function adds movie dictionary to user_data and returns user_data
+    '''
+    user_data["watched"] = [movie]

```

Comment:
Careful not to overwrite the original data!

By creating a new list and assigning it to `user_data`, you replace all of the original data with this new list that only has a single movie in its list.

Think about how you could update original data rather than replacing it.